### PR TITLE
Add `requestorAppUrl`

### PR DESCRIPTION
**Pull request by descorp**
# Open OPR

## Changes

* Now it is possible to set `threeDS2.requestorAppUrl` to control OOB flow on native 3DS2

### CODE REVIEW

File: android/src/main/java/com/adyenreactnativesdk/configuration/ThreeDSConfigurationParser.kt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            this.config = config.getMap(THREEDS2_KEY)!!
+        } else {
+            this.config = config
+        }
+    }
+
+    val requestorAppUrl: String?
+        get() = if (config.hasKey(THREEDS2_REQUESTOR_APP_URL_KEY)) {
+            config.getString(THREEDS2_REQUESTOR_APP_URL_KEY)
+        } else null
```
Comment (Robert-SD) [inline android/src/main/java/com/adyenreactnativesdk/configuration/ThreeDSConfigurationParser.kt:24]:
```suggestion
        } else {
         null
        }
```
You can decide if we should add curly braces for this case as well. The other single lines have them as well.

File: android/src/test/java/com/adyenreactnativesdk/configuration/ThreeDSConfigurationParserTest.kt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+class ThreeDSConfigurationParserTest {
+
+    @Test
+    fun testApplyThreeDSConfiguration() {
+        val mockBuilder = mock(Adyen3DS2Configuration.Builder::class.java)
+        val config = WritableMapMock()
+        config.putString(ThreeDSConfigurationParser.THREEDS2_REQUESTOR_APP_URL_KEY , "https://testing.com")
+
+        // WHEN
+        val sut = ThreeDSConfigurationParser(config)
```
Comment (Robert-SD) [inline android/src/test/java/com/adyenreactnativesdk/configuration/ThreeDSConfigurationParserTest.kt:17]: Shall we rename sut?

**Review by Robert-SD — DISMISSED** [review]

File: android/src/main/java/com/adyenreactnativesdk/configuration/ThreeDSConfigurationParser.kt (hunk 1 above)
Comment (descorp) [reply android/src/main/java/com/adyenreactnativesdk/configuration/ThreeDSConfigurationParser.kt:26]: I'll do full formatting in another PR :)

Comment (descorp) [reply android/src/main/java/com/adyenreactnativesdk/configuration/ThreeDSConfigurationParser.kt:24]: Updated [bda2158](https://github.com/Adyen/adyen-react-native/pull/430/commits/bda215848c14dd61c0273291e48518f0b971e0c0)

**Review by Robert-SD — APPROVED** [review]

**Pull request merged**